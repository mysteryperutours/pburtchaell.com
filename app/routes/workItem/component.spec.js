import React from 'react';
import WorkItemRoute from './component';

describe(WorkItemRoute.name, () => {
  it('should render component', () => {
    expect(shallow(<WorkItemRoute />)).toMatchSnapshot();
  });
});
