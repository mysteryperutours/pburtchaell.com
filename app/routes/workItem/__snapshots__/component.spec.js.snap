exports[`WorkItemRoute should render component 1`] = `
<RouteContainer>
  <article
    role="article" />
</RouteContainer>
`;
